packages/fxa-auth-server/lib/customs.js

```diff
@@ -4,192 +4,150 @@
 
 'use strict';
 
-const Joi = require('joi');
-const createBackendServiceAPI = require('./backendService');
-const { config } = require('../config');
+import axios from 'axios';
+import { config } from '../config';
 const localizeTimestamp =
   require('../../../libs/shared/l10n/src').localizeTimestamp({
     supportedLanguages: config.get('i18n').supportedLanguages,
     defaultLanguage: config.get('i18n').defaultLanguage,
   });
 const serviceName = 'customs';
 
-module.exports = function (log, error, statsd) {
-  const CustomsAPI = createBackendServiceAPI(
-    log,
-    config,
-    serviceName,
-    {
-      check: {
-        path: '/check',
-        method: 'POST',
-        validate: {
-          payload: {
```

Comment:
Having this validation is nice but I think overkill atm since auth-server talks to less services via an API.